lib/main.dart

```diff
@@ -274,6 +278,14 @@ class _TenTenOneState extends State<TenTenOneApp> {
     try {
       await setupRustLogging();
 
+      PackageInfo packageInfo = await PackageInfo.fromPlatform();
+      String version = packageInfo.version;
+
+      FLog.info(text: "TenTenOne Version:" + version);
```

Comment:
🙃 We might want a space here.

```suggestion
      FLog.info(text: "TenTenOne Version: " + version);
```